LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+(C) Copyright [2022] Hewlett Packard Enterprise Development LP
```

Comment:
It should be 2020,2022", don't drop old years.

Reply:
Fixed.